charts/memgraph-lab/README.md

```diff
@@ -0,0 +1,51 @@
+## Memgraph Lab Standalone Kubernetes Helm Chart
+A Helm Chart for deploying standalone Memgraph Lab on Kubernetes.
+
+## Installing the Memgraph Lab Helm Chart
+To install the Memgraph Lab Helm Chart, follow the steps below:
+```
+helm install <release-name> memgraph/memgraph-lab
+```
+Replace `<release-name>` with a name of your choice for the release.
+
+## Changing the default chart values
+To change the default chart values, run the command with the specified set of flags:
+```
+helm install <resource-name> memgraph/memgraph --set <flag1>=<value1>,<flag2>=<value2>,...
```

Comment:
This should probably be `memgraph/memgraph-lab` ?

Reply:
yep, updated in latest commit.